### ISSUE TITLE
Wine sandbox on non english system

### ISSUE THREAD

**Issue by G4E** [lutris/lutris#2292]
Hello,
By default, the option "Create a sandbox for wine folders" is ticked when we create a new prefix, by creating a new game with a wine runner.
It doesn't seems to work on my system, as I have found some data in my ~/Documents folder created by some app in a wine prefix.
If I go in the wine's prefix/drive_c/users/tom
I can see that indeed, wine has created some symlinks to my home folders :
```
ls -l
total 88
drwxr-xr-x 3 tom tom 4096 14 août  22:19  AppData/
drwxr-xr-x 3 tom tom 4096 14 août  22:19 'Application Data'/
lrwxrwxrwx 1 tom tom   17 14 août  22:19  Bureau -> /home/tom/Desktop/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Contacts/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Cookies/
drwxr-xr-x 2 tom tom 4096 18 août  22:30  Desktop/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Downloads/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Favorites/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Links/
drwxr-xr-x 5 tom tom 4096 14 août  22:19 'Local Settings'/
lrwxrwxrwx 1 tom tom   17 14 août  22:19 'Ma musique' -> /home/tom/Musique/
lrwxrwxrwx 1 tom tom   19 14 août  22:19 'Mes documents' -> /home/tom/Documents/
lrwxrwxrwx 1 tom tom   16 14 août  22:19 'Mes images' -> /home/tom/Images/
lrwxrwxrwx 1 tom tom   17 14 août  22:19 'Mes vidéos' -> /home/tom/Vidéos/
drwxr-xr-x 2 tom tom 4096 18 août  22:30 'My Documents'/
drwxr-xr-x 2 tom tom 4096 18 août  22:30 'My Music'/
drwxr-xr-x 2 tom tom 4096 18 août  22:30 'My Pictures'/
drwxr-xr-x 2 tom tom 4096 18 août  22:30 'My Videos'/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  NetHood/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  PrintHood/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Recent/
drwxr-xr-x 2 tom tom 4096 14 août  22:19 'Saved Games'/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Searches/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  SendTo/
drwxr-xr-x 3 tom tom 4096 14 août  22:19 'Start Menu'/
drwxr-xr-x 2 tom tom 4096 14 août  22:52  Temp/
drwxr-xr-x 2 tom tom 4096 14 août  22:19  Templates/
```
If I force a folder in the "Sandbox directory" option, something strange happens :
```
ls -l
total 72
drwxr-xr-x 3 tom tom 4096 27 août  22:12  AppData/
drwxr-xr-x 3 tom tom 4096 27 août  22:12 'Application Data'/
lrwxrwxrwx 1 tom tom   17 27 août  22:12  Bureau -> /home/tom/Desktop/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Contacts/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Cookies/
lrwxrwxrwx 1 tom tom   34 27 août  22:12  Desktop -> /home/tom/Games/sandbox/Desktop/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Downloads/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Favorites/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Links/
drwxr-xr-x 5 tom tom 4096 27 août  22:12 'Local Settings'/
lrwxrwxrwx 1 tom tom   17 27 août  22:12 'Ma musique' -> /home/tom/Musique/
lrwxrwxrwx 1 tom tom   19 27 août  22:12 'Mes documents' -> /home/tom/Documents/
lrwxrwxrwx 1 tom tom   16 27 août  22:12 'Mes images' -> /home/tom/Images/
lrwxrwxrwx 1 tom tom   17 27 août  22:12 'Mes vidéos' -> /home/tom/Vidéos/
lrwxrwxrwx 1 tom tom   39 27 août  22:12 'My Documents' -> '/home/tom/Games/sandbox/My Documents'/
lrwxrwxrwx 1 tom tom   35 27 août  22:12 'My Music' -> '/home/tom/Games/sandbox/My Music'/
drwxr-xr-x 2 tom tom 4096 27 août  22:12 'My Pictures'/
lrwxrwxrwx 1 tom tom   36 27 août  22:12 'My Videos' -> '/home/tom/Games/sandbox/My Videos'/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  NetHood/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  PrintHood/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Recent/
drwxr-xr-x 2 tom tom 4096 27 août  22:12 'Saved Games'/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Searches/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  SendTo/
drwxr-xr-x 3 tom tom 4096 27 août  22:12 'Start Menu'/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Temp/
drwxr-xr-x 2 tom tom 4096 27 août  22:12  Templates/

```
As you can see, wine's symlink are still there **in french**, but Lutris created symlinks for those folders too, **in english**.
TLDR :
Wine follow LOCAL to create its symlinks, Lutris doesn't, so sandboxing doesn't work on non english locals.
Thanks.

**Comment by G4E** [comment]
Why do you have a specific [whitelist](https://github.com/lutris/lutris/blob/master/lutris/util/wine/prefix.py#L8) for those folders, and not simply unsymlink them like you attempt to do [here](https://github.com/lutris/lutris/blob/master/lutris/util/wine/prefix.py#L108-112) ?

**Comment by strycore** [comment]
Thanks for catching that last bug, I've fixed that as well. I've added support for multiple languages for Wine folders, right now only English and French are supported.

**Comment by G4E** [comment]
Alternatively, those folders are defined by the registry keys in 
```
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders
```
Keys :
```
Desktop
My Music
My Pictures
My Videos
Personal (for My Documents)
```
So you could read those keys to populate your dictionary, and it would be langage agnostic ?
I can try to make a pull request later if you don't have time.

**Comment by G4E** [comment]
I opened a pull request (#2300 ) to show you a way to do it
